internal/rdc/client.go

```diff
@@ -152,12 +155,18 @@ func (c *Client) PollJob(ctx context.Context, id string, interval, timeout time.
 	deathclock := time.NewTimer(timeout)
 	defer deathclock.Stop()
 
+	var retryTime int
```

Comment:
Time is time. Count is count :)
```suggestion
	var retryCount int
```

or

```suggestion
	var attempt int
```